mage_ai/api/resources/PipelineResource.py

```diff
@@ -20,6 +20,22 @@ async def collection(self, query, meta, user, **kwargs):
         if include_schedules:
             include_schedules = include_schedules[0]
 
+        pipeline_types = query.get('type[]', [None])
```

Comment:
@dy46 Does the logic and updates in this file (`mage_ai/api/resources/PipelineResource.py`) in particular seem okay? The rest of the PR is FE changes.

Reply:
can you just set the default here to be an empty array? then you don't have to worry about there being a `None` element

Reply:
I cleaned it up a bit in the latest commit: https://github.com/mage-ai/mage-ai/pull/2187/commits/31df5b3304fd016bb9c5501610fae278d6eefb05